docs/source/ecf.rst

```diff
@@ -767,12 +767,16 @@ Used by exotic-ld if compute_ld=True. The fully qualified path to the directory
 
 exotic_ld_grid
 ''''''''''''''
-Used by exotic-ld if compute_ld=True. 1D or 3D model grid.
+Used by exotic-ld if compute_ld=True. 1D or 3D model grid, or "custom" for a custom stellar intensity grid.
```

Comment:
@kevin218, just an FYI that we'll need to be careful when merging the code from this PR together your #609 later since I think both PRs change the use of this parameter